### ISSUE TITLE
Local Trade Support

### ISSUE THREAD

**Issue by leo60228** [olliz0r/Ledybot#15]
I know this would be a lot of work to implement, but it would be helpful if this supported local trades too. I would help code it if I knew where to start.

**Comment by Bappsack1994** [comment]
do u mean Link Trades?
If yes we need to Add Friendcodes auto., request/accept Trades from Persons and i just want to say it will be hard to let the bot handle human Stupidness :P

**Comment by leo60228** [comment]
Yes. Sorry for not replying! I know C++ and have some RE ebooks but I don't have a clue how to figure out what the addresses and functions are.

**Comment by Bappsack1994** [comment]
u need to search some addresses for your project.

1. u need a Detection for receiving a Link Trade in Plaza or send a Link Trade to a Player.
2. Some Addresses for Detection Menu/In Menus (Yes, Trade, blablabla).
3. Actual in Box Setup Routine, select(Inject? Pokemon), trade, after finish leave or repeat.
(4). Maybe a Way to inject a Friendcode into the 3DS Process that holds the friends? iono
